src/plugins/unified_field_list/public/hooks/use_query_subscriber.ts

```diff
@@ -24,6 +24,7 @@ export interface QuerySubscriberParams {
 export interface QuerySubscriberResult {
   query: Query | AggregateQuery | undefined;
   filters: Filter[] | undefined;
+  timeRange: TimeRange | undefined;
```

Comment:
@jughosta I added `timeRange` to the `QuerySubscriberResult` to provide easy access to the relative range in Discover -- hope this change is ok!

Reply:
There is already `fromDate` and `toDate` which are based on a correct "now" point. Why not to reuse it?

Reply:
Unified Histogram needs access to a relative time range in order to support the edit visualization button. Otherwise we would always pass an absolute time range when navigating to Lens which isn't the desired behaviour. Do you think it would be better to pass in `timeRange` plus `fromDate` and `toDate`? Another alternative could be to pass in `timeRange` and `nowDate` to ensure an identical `now`.

Reply:
Would it be possible to call `timefilter.timefilter.getTime()` directly when Edit visualization button is pressed?

Reply:
Ideally we're trying to avoid hard dependencies on things like UI components and shared state to allow more flexibility and use cases, but I think a good alternative could be to accept something like a `getEditVisualizationTimeRange` callback that defaults to `timefilter.timefilter.getTime`, since I think that would cover all use cases. Then I could switch back to the absolute timeRange for the `timeRange` prop.

Reply:
I reverted the changes to `useQuerySubscriber` and added a `getRelativeTimeRange` callback to Unified Histogram instead.